solr/core/src/test/org/apache/solr/client/solrj/impl/ConnectionReuseTest.java

```diff
@@ -83,7 +83,8 @@ private SolrClient buildClient(CloseableHttpClient httpClient, URL url) {
     switch (random().nextInt(3)) {
       case 0:
         // currently, only testing with 1 thread
-        return new ConcurrentUpdateSolrClient.Builder(url.toString() + "/" + COLLECTION)
+        return new ConcurrentUpdateSolrClient.Builder(url.toString())
```

Comment:
sigh...   if only a builder new what a url was and we didn't .tostring everywhere.  ;-)   #beatdeadhorse